include/capypdf.hpp

```diff
@@ -129,14 +129,18 @@ class DrawContext : public CapyC<CapyPDF_DrawContext> {
     friend class Generator;
 
 public:
+    void cmd_cm(double a, double b, double c, double d, double e, double f) { CAPY_CPP_CHECK(capy_dc_cmd_cm(*this, a, b, c, d, e, f)); }
     void cmd_f() { CAPY_CPP_CHECK(capy_dc_cmd_f(*this)); }
     void cmd_re(double x, double y, double w, double h) {
         CAPY_CPP_CHECK(capy_dc_cmd_re(*this, x, y, w, h));
     }
     void cmd_rg(double r, double g, double b) { CAPY_CPP_CHECK(capy_dc_cmd_rg(*this, r, g, b)); }
 
+    void set_custom_page_properties(PageProperties const &props) {
+        capy_dc_set_custom_page_properties(*this, props);
```

Comment:
Sorry I missed this earlier, but all calls to the C API need to be wrapped in `CAPY_CPP_CHECK`.